orangecontrib/spectroscopy/widgets/owspectra.py

```diff
@@ -668,6 +665,13 @@ class CurvePlot(QWidget, OWComponent, SelectionGroupMixin):
     range_x2 = Setting(None)
     range_y1 = Setting(None)
     range_y2 = Setting(None)
+    prominence = Setting(None, schema_only=True)
+    Line_Overlap = Setting(None, schema_only=True)
+    peak_min = Setting(None, schema_only=True)
+    peak_max = Setting(None, schema_only=True)
+    peak_locations = Setting([], schema_only=True)
+    minimum_point = Setting(None, schema_only=True)
```

Comment:
minimum_point, maximum_points should not be a setting, because they are always defined by data and never changed by the UI.